specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/iotSecuritySolutions.json

```diff
@@ -481,86 +483,106 @@
               },
               {
-                "value": "IdenticalAuthenticationCredentials",
-                "description": "IoT Hub authentication credentials are used by multiple devices. This may indicate an illegitimate device impersonating a legitimate device. Duplicate credential use increases the risk of device impersonation by a malicious actor."
+                "value": "IoT_PermissiveFirewallPolicy",
+                "description": "An Allowed firewall policy was found (INPUT/OUTPUT). The policy should Deny all traffic by default and define rules to allow necessary communication to/from the device."
               },
               {
-                "value": "DenyDefaultIpPolicy",
-                "description": "IP filter configuration should have rules defined for allowed traffic, and should by default, deny all other traffic by default."
+                "value": "IoT_PermissiveInputFirewallRules",
+                "description": "A rule in the firewall has been found that contains a permissive pattern for a wide range of IP addresses or Ports."
               },
               {
-                "value": "TooLargeIPRange",
-                "description": "An allow IP filter rule source IP range is too large. Overly permissive rules can expose your IoT hub to malicious actors."
+                "value": "IoT_PermissiveOutputFirewallRules",
+                "description": "A rule in the firewall has been found that contains a permissive pattern for a wide range of IP addresses or Ports."
               },
               {
-                "value": "EnableDiagnosticsLog",
-                "description": "Enable logs and retain them for up to a year. Retaining logs enables you to recreate activity trails for investigation purposes when a security incident occurs or your network is compromised."
+                "value": "IoT_PrivilegedDockerOptions",
+                "description": "Edge module is configured to run in privileged mode, with extensive Linux capabilities or with host-level network access (send/receive data to host machine)."
+              },
+              {
+                "value": "IoT_SharedCredentials",
+                "description": "Same authentication credentials to the IoT Hub used by multiple devices. This could indicate an illegitimate device impersonating a legitimate device. It also exposes the risk of device impersonation by an attacker."
+              },
+              {
+                "value": "IoT_VulnerableTLSCipherSuite",
+                "description": "Insecure TLS configurations detected. Immediate upgrade recommended."
               }
             ]
           }
         },
-        "description": {
+        "name": {
           "type": "string",
           "readOnly": true
         },
         "status": {
           "type": "string",
           "enum": [
-            "TurnedOff",
-            "TurnedOn"
+            "Disabled",
```

Comment:
Again, this would be classified as a breaking change. Older versions of the SDK clients will be unable to send to or read from the server if the contract is changed.

Reply:
We do not have old clients. the API was not published yet for customers

Reply:
Thanks for confirming